string measurment
string sensor
float64 value